test/Handler/CommentsSpec.hs

```diff
@@ -46,17 +54,18 @@ spec = withApp $ do
 
     describe "POST CommentsR" $ do
         it "does not allow unauthenticated commenting" $ do
-            Entity siteId _ <- createSite
+            siteId <- runDB $ insert $ buildSite
 
             post $ CommentsR siteId
 
             statusIs 401
 
         it "allows commenting by authenticated users" $ do
-            u <- createUser "1"
-            Entity siteId _ <- createSite
+            (siteId, user) <- runDB $ (,)
```

Comment:
Look at you, naming your variables! Why did `u` become unacceptable here but not elsewhere?